### DISCUSSION TITLE
Creating resources using C# is impossible.

### DISCUSSION THREAD

**Discussion by Ghostinoo** [citizenfx/fivem#1789] [Engineering Discussion]
Programming in C# using the standard CitizenFX framework is practically impossible. I think I am not the only one who has tried for a long time to create a resource in C# without even being able to get the Player object on the server side. The Docs are unreadable, unclear, with no reference to classes and their structure. Moreover, I have found them completely lacking in examples. I hope you will consider this appeal because your project is very beautiful and, despite its discreet age, it remains promising.

**Comment by thorium-cfx** [comment]
I assume this is a request for the improvements of the runtime, but written in a way to put responsibility and blame on us to think and develop solutions for it, just to suit your needs?

To get such discussions started; set up a case (or multiple), think of possible solutions and be open to discuss and change your own mind. There are many C# developers I work with that don't experience said issues and do in fact come with interesting improvements to the platform of the things they do come across. You're welcome to be one of them.
